ui/app/src/components/slippagePanel/Index.vue

```diff
@@ -0,0 +1,155 @@
+
+
+<template>
+  <div class="slippage-section mh-70">
+    <div class="d-flex fd-row">
+      <p class="text--small text--italic text--normal" align="left">
+        Slippage tolerance
+      </p>
+    </div>
+    <div class="field-wrappers d-flex fd-row j-start">
+      <div class="slippage-item" v-bind:class="{ 'slippage-item__selected': localSlippage === '0.5' }" @click="handleUpdateSlippage('0.5')">
+        <p class="text--standard text--normal text--regular">
+          0.5%
+        </p>
+      </div>
+      <div class="slippage-item" v-bind:class="{ 'slippage-item__selected': localSlippage === '1.0' }" @click="handleUpdateSlippage('1.0')">
+        <p class="text--standard text--normal text--regular">
+          1.0%
+        </p>
+      </div>
+      <div class="slippage-item" v-bind:class="{ 'slippage-item__selected': localSlippage === '2.0' }" @click="handleUpdateSlippage('2.0')">
+        <p class="text--standard text--normal text--regular">
+          2.0%
+        </p>
+      </div>
+      <div class="sif-input__wrapper custom-slippage-input-container">
+        <input
+          @click="$event.target.select()"
+          @input="validateSlippage"
+          v-model="localSlippage"
+          class="sif-input num_percent"
+          type="number"
+          step="any"
+          min="0"
+        >
+        <span>%</span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import { computed } from "@vue/reactivity";
+export default defineComponent({
+  components: { },
+  props: {
+    slippage: { type: String, default: "0" },
+  },
+  emits: [
+    "update:slippage",
+  ],
+  setup(props, context) {
+    const localSlippage = computed({
+      get: () => props.slippage,
+      set: (amount) => handleUpdateSlippage(amount),
+    });
+    function handleUpdateSlippage(amount: string) {
+      context.emit("update:slippage", amount);
```

Comment:
Nice.

nit: I noticed in the screen recording that the display decimal point used a comma, not period to separate

Reply:
@59023g yes, same for other inputs. Should we use a dot for all of them?

Reply:
I would. But not a dealbreaker